Rsk.Samples.IdentityServer4.AdminUiIntegration/Startup.cs

```diff
@@ -137,6 +138,26 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             services.AddMvc();
+            services.AddMvcCore()
+                .AddAuthorization();
+            
+            services.AddAuthentication("Bearer")
+                .AddIdentityServerAuthentication(options =>
```

Comment:
**improvement:** can you switch this out for the JWT bearer auth handler & reference token auth handler? This library is now obsolete: https://leastprivilege.com/2020/07/06/flexible-access-token-validation-in-asp-net-core/

Reply:
Okay - I lifted this from our Webhooks GitHub project so we should update that too at some point

Reply:
Well volunteered.